### PR TITLE
New package: hirosystems.chainhook version 1.6.1

### PR DESCRIPTION
**Pull request by MicaiahReid**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/153728)

### CODE REVIEW

**Comment by MicaiahReid** [comment]
@microsoft-github-policy-service agree company="Hiro"

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\chainhook\bin\chainhook.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 881.)

File: manifests/h/HiroSystems/Chainhook/1.6.1/HiroSystems.Chainhook.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+InstallerType: wix
+Scope: machine
+UpgradeBehavior: install
+Commands:
+- chainhook
+ProductCode: '{A8AB46B7-5472-4FDE-B4D5-6E58CE6F35D0}'
+ReleaseDate: 2024-05-15
+AppsAndFeaturesEntries:
+- UpgradeCode: '{E41901A3-0B71-4D86-9175-B12D7C7533BE}'
+Installers:
```
Comment (stephengillie) [inline manifests/h/HiroSystems/Chainhook/1.6.1/HiroSystems.Chainhook.installer.yaml:16]:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
With the dependency installed, this application doesn't seem to have any output: 
image

Is this expected?

Verify: 
image

**Comment by MicaiahReid** [comment]
Hi @stephengillie, thanks for the review.

That's interesting, do you get any result when running `./chainhook.exe -V`?

**Comment by stephengillie** [comment]
The `-v` switch also gives no output:
image

**Comment by MicaiahReid** [comment]
> Hello @MicaiahReid,
> 
> This pull request has been automatically marked as stale because it has been marked as requiring author feedback but has not had any recent activity. It will be closed if no further activity occurs **within 3 days of this comment**.
> 
> Template: msftbot/noRecentActivity

I am investigating why there is no result from running the tool on windows and will update this PR once completed.

**Pull request closed without merging**